src/api-engine/api/models.py

```diff
@@ -22,7 +22,7 @@
     LogLevel,
     HostType,
     K8SCredentialType,
-    separate_upper_class,
+    separate_upper_class, 
```

Comment:
Do we need to add this empty space?

Reply:
no necessary, I have removed it in the last commit.